README.md

```diff
@@ -1 +1,204 @@
-# mathMagicians
\ No newline at end of file
+<a name="readme-top"></a>
+
+<div align="center">
+
+  <img src="logo.png" alt="logo" width="140"  height="auto" />
+  <br/>
+
+  <h3><b></b></h3>
+
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ](#faq)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 Math Magicians <a name="about-project"></a>
+
+> "Math magicians" is a website for all fans of mathematics. It is a Single Page App (SPA) that allows users to: make simple calculations, read a random math-related quote. By building this application,I will learn how to use React.
+
+**[your_project__name]** is a...
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+> Describe the tech stack and include only the relevant sections that apply to your project.
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://reactjs.org/">React.js</a></li>
+  </ul>
+</details>
+
+
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+>  Key features of the application.
+
+- **Learn how to use proper ES6 syntax.**
+- **Use ES6 modules to write modular JavaScript.**
+- **Follow Gitflow.**
+- **Handle events in a React app.**
+- **Use React life cycle methods.**
+- **Understand the mechanism of lifting state up.**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+>
+
+- [Live Demo Link](https://github.com/Eldonratzlaff/mathMagicians)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+> Clone the repository by clicking on the 'Code' button and copy the link
+
+To get a local copy up and running
+
+### Prerequisites
+
+In order to run this project you need:
+
+-visual studio
+-github
+
+### Setup
+
+Clone this repository to your desired folder:
+
+
+### Install
+
+Install this project with: ```
+  npm install
+```
```

Comment:
- [ ] Kindly put these on the same line to avoid breaking of display of the contents below. Change this to 
![image](https://user-images.githubusercontent.com/94551484/222808443-274cacfb-823e-4bd0-8ec9-dd2e138f9325.png)